examples/qwik/src/routes/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { component$, useServerMount$, useSignal } from '@builder.io/qwik'
+import { grafbaseClient } from '~/utils/grafbase'
+
+interface Plants {
+  plantCollection: {
+    edges: Plant[]
+  }
+}
+
+interface Plant {
+  id: string
+  name: string
+  description: string
+}
+
+export const GetAllPlantsQuery = `
+  query GetAllPlants($first: Int!) {
+    plantCollection(first: $first) {
+      edges {
+        node {
+          id
+          name
+          description
+        }
+      }
+    }
+  }
+`
+
+export const AddNewPlantMutation = `
+  mutation AddNewPlant($name: String!, $description: String!) {
+    plantCreate(input: { name: $name, description: $description }) {
+      plant {
+        id
+        name
+        description
+      }
+    }
+  }
+`
+
+export default component$(() => {
+  const newPlant = useSignal('')
+  const newPlantDescription = useSignal('')
+  const allPlants = useSignal<Plants>()
+
+  useServerMount$(async () => {
+    const plants: Plants = await grafbaseClient({
```

Comment:
We could for the purposes of this example ditch the util and just use `fetch` here since we only use it once:

```ts
 const {
      data: { messageCollection }
    }: {
      data: { messageCollection: { edges: { node: Message }[] } }
    } = await fetch(import.meta.env.VITE_GRAFBASE_API_URL as string, {
      method: 'POST',
      headers: {
        'content-type': 'application/json'
      },
      body: JSON.stringify({
        query: GetAllMessagesQuery,
        variables: {
          first: 100
        }
      })
    }).then((res) => res.json())
```